fiftyone/server/aggregations.py

```
@@ -177,15 +177,40 @@ async def aggregate_resolver(
         for result in results:
             if isinstance(result, RootAggregation):
-                result.slice = await slice_view._async_aggregate(foa.Count())
+                result.slice = groups
+                result.count = samples
+                result.exists = samples
                 break
 
     return results
 
 
+async def _grouped_root_counts(view, maxTimeMS=None):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_grouped_root_counts`

(ANN202)